service/src/main/java/uk/nhs/adaptors/gp2gp/mhs/MhsRequestBuilder.java

```diff
@@ -46,6 +46,8 @@
         return uri
             .accept(MediaType.APPLICATION_JSON)
             .header(INTERACTION_ID, MHS_OUTBOUND_INTERACTION_ID)
+            .header("wait-for-response", "false")
```

Comment:
Can we create constants for these like above? Instead of "false" we could place "Boolean.FALSE.toString()".